rotkehlchen/chain/evm/decoding/cowswap/decoder.py

```diff
@@ -370,6 +370,10 @@ def _decode_normal_claim(self, context: DecoderContext) -> DecodingOutput:
         amount = asset_normalized_value(amount=raw_amount, asset=self.vcow_token)
         airdrop_identifier: Literal['cow_mainnet', 'cow_gnosis'] = 'cow_mainnet' if self.evm_inquirer.chain_id == ChainID.ETHEREUM else 'cow_gnosis'  # noqa: E501
         for event in context.decoded_events:
+            if event.event_type == HistoryEventType.SPEND and event.counterparty != 'gas':
+                event.event_subtype = HistoryEventSubType.PAYMENT
+                event.notes = f'Pay {event.balance.amount} {event.asset.symbol_or_name()} to claim vCOW'  # noqa: E501
+                continue
```

Comment:
notice that below the code that you added there is a break statment. I believe that the vcow receive always happens after the payment but you should add a comment stating this. Someone that is not aware could change the code and break the assumption here